tasks/main.yml

```diff
@@ -9,17 +9,17 @@
   get_url: url={{ item.url }} dest={{ instantclient_oracle_package_dest }}/{{ item.filename }}
            sha256sum={{ item.sha256 }}
   when: item.url is defined
-  with_items: instantclient_oracle_packages
+  with_items: "{{instantclient_oracle_packages}}"
 
```

Comment:
Spaces inside of brackets, i.e. {{ foo }} instead of {{foo}}. Same on lines 18 and 22.